ngraph/core/reference/include/ngraph/runtime/reference/interpolate.hpp

```diff
@@ -226,6 +222,16 @@ namespace ngraph
                     , m_out_shape{out_shape}
                     , m_scales{scales}
                 {
+                    size_t input_rank = input_data_shape.size();
+                    std::vector<float> all_scales(input_rank, 1.0f);
```

Comment:
why we need this local vector? Looks like we can write data to `m_all_scales` without temporary vectors.

Reply:
Done.